### PR TITLE
i18n(es): Translated `contribute.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `contribute.mdx` to spanish 🚀

### CODE REVIEW

File: src/content/docs/es/contribute.mdx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+title: Contribuye a la documentación
+description: Una guía para contribuir a la documentación de Astro.
+i18nReady: true
+---
+import Button from '~/components/Button.astro'
+import ContributorList from '~/components/ContributorList.astro'
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+
+Aceptamos contribuciones de cualquier tamaño y contribuyentes de cualquier nivel de habilidad. Como un proyecto de código abierto, creemos en devolver a nuestros contribuyentes. Estamos encantados de ayudar con orientación sobre PR, escritura técnica y convertir cualquier idea de función en realidad.
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/contribute.mdx:11]:
```suggestion
Aceptamos contribuciones de cualquier tamaño y contribuyentes de todos los niveles de habilidad. Como un proyecto de código abierto, creemos en devolver a nuestros contribuyentes. Estamos encantados de ayudar con orientación sobre PRs, escritura técnica y convertir cualquier en realidad.
```

File: src/content/docs/es/contribute.mdx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## Maneras de contribuir
+
+Hay muchas formas de contribuir al sitio web de la documentación de Astro!
+
+El sitio web de la documentación de Astro es ... ¡un sitio web de Astro! Mantenerlo requiere no solo contenido escrito, sino también mantener el código de Astro y abordar preocupaciones de a11y, CSS, UI y UX. También hacemos que nuestra documentación esté disponible en varios idiomas, por lo que necesitamos ayuda para traducir todo el sitio.
+
+¡También puedes hacer una contribución enorme al involucrarte dejando comentarios de revisión en PR, agregando ideas en problemas y discusiones existentes de GitHub y participando en nuestras tareas de mantenimiento de problemas "Fijados"!
+
+Cada PR, especialmente los PR de traducción, ¡necesita evaluadores! Revisar PR y dejar comentarios, sugerencias o un aprobador "LGTM!" ("¡Me parece bien!") Es una gran manera de comenzar en Team Docs y aprender más sobre Astro.
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/contribute.mdx:21]:
```suggestion
Cada PR, especialmente las PR de traducción, ¡necesita evaluadores! Revisar la PR y dejar comentarios, sugerencias o un aprobador "LGTM!" ("¡Me parece bien!") Es una gran manera de comenzar en Team Docs y aprender más sobre Astro.
```

File: src/content/docs/es/contribute.mdx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+- [Guía principal para contribuir](https://github.com/withastro/docs/blob/main/CONTRIBUTING.md) - ¡Por favor consulta esto primero!
+- [Guía de Redacción](https://github.com/withastro/docs/blob/main/WRITING.md) - Consejos para escribir en el estilo de la documentación de Astro.
+- [Guía de Traducción](https://github.com/withastro/docs/blob/main/TRANSLATING.md)  - Información sobre cómo ayudar a traducir la documentación de Astro a tu idioma.
+- [Guía de Recetas](https://github.com/withastro/docs/blob/main/RECIPES.md) - Aprende sobre cómo escribir y contribuir con una receta.
+
+## Guías de la comunidad
+
+Estas son las guías para los roles dentro de la comunidad de Astro.
+- [Guía del Mantenedor](https://github.com/withastro/docs/blob/main/MAINTAINERS.md) - Como mantener este repositorio como un mantenedor de Astro.
+- [Guía de Escuadrón de Apoyo](https://github.com/withastro/docs/blob/main/SUPPORTSQUAD.md) - Como ayudar a Team Docs como parte de su rol de soporte.
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/contribute.mdx:36]:
```suggestion
- [Guía de Escuadrón de Apoyo](https://github.com/withastro/docs/blob/main/SUPPORTSQUAD.md) - Como ayudar a Team Docs como parte de soporte técnico.
```

**Review by kevinzunigacuellar — COMMENTED** [review]
Algunos comentarios, gracias @dreyfus92 💜

**Comment by dreyfus92** [comment]
Muchas gracias por el review @kevinzunigacuellar 😁

File: src/content/docs/es/contribute.mdx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+title: Contribuye a la documentación
+description: Una guía para contribuir a la documentación de Astro.
+i18nReady: true
+---
+import Button from '~/components/Button.astro'
+import ContributorList from '~/components/ContributorList.astro'
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+
+Aceptamos contribuciones de cualquier tamaño y contribuyentes de todos los niveles de habilidad. Como un proyecto de código abierto, creemos en devolver a nuestros contribuyentes. Estamos encantados de ayudar con orientación sobre PRs, escritura técnica y convertir cualquier idea de función en realidad.
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/contribute.mdx:11]:
```suggestion
Aceptamos contribuciones de cualquier tamaño y contribuyentes de todos los niveles de habilidad. Como un proyecto de código abierto, creemos en devolver a nuestros contribuyentes. Estamos encantados de ayudar con orientación sobre PRs, escritura técnica y convertir cualquier idea en realidad.
```

File: src/content/docs/es/contribute.mdx [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+## Maneras de contribuir
+
+Hay muchas formas de contribuir al sitio web de la documentación de Astro!
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/contribute.mdx:15]:
```suggestion
Hay muchas formas de contribuir a la documentación de Astro!
```

File: src/content/docs/es/contribute.mdx [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+El sitio web de la documentación de Astro es ... ¡un sitio web de Astro! Mantenerlo requiere no solo contenido escrito, sino también mantener el código de Astro y abordar preocupaciones de a11y, CSS, UI y UX. También hacemos que nuestra documentación esté disponible en varios idiomas, por lo que necesitamos ayuda para traducir todo el sitio.
+
+¡También puedes hacer una contribución enorme al involucrarte dejando comentarios de revisión en PR, agregando ideas en problemas y discusiones existentes de GitHub y participando en nuestras tareas de mantenimiento de problemas "Fijados"!
+
+Cada PR, especialmente las PR de traducción, ¡necesita evaluadores! Revisar la PR y dejar comentarios, sugerencias o un aprobador "LGTM!" ("¡Me parece bien!") Es una gran manera de comenzar en Team Docs y aprender más sobre Astro.
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/contribute.mdx:21]:
```suggestion
Cada PR, especialmente las PRs de traducción, ¡necesita evaluadores! Revisar la PR y dejar comentarios, sugerencias o aprobarla con un "LGTM!" ("¡Me parece bien!") Es una gran manera de comenzar en Team Docs y aprender más sobre Astro.
```

File: src/content/docs/es/contribute.mdx [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+## Guías de la comunidad
+
+Estas son las guías para los roles dentro de la comunidad de Astro.
+- [Guía del Mantenedor](https://github.com/withastro/docs/blob/main/MAINTAINERS.md) - Como mantener este repositorio como un mantenedor de Astro.
+- [Guía de Escuadrón de Apoyo](https://github.com/withastro/docs/blob/main/SUPPORTSQUAD.md) - Como ayudar a Team Docs como parte de soporte técnico.
+
+## Nuestros contribuyentes
+
+Estos documentos son traídos por toda esta gente útil. [¡Únete a nosotros en GitHub!](https://github.com/withastro/docs)
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/contribute.mdx:40]:
```suggestion
Estos documentos son traídos por nuestra comunidad. [¡Únete a nosotros en GitHub!](https://github.com/withastro/docs)
```

**Review by kevinzunigacuellar — COMMENTED** [review]
Segunda re-leida, gracias por los fixes @dreyfus92

**Review by Yan-Thomas — CHANGES_REQUESTED** [review]
Hey @dreyfus92, just found out now that we're missing the `ui.ts` label for the "Contributor Guides" item in the sidebar that is were you go to this page, could you add this to the PR? After this LGTM & good to be merged!

**Comment by dreyfus92** [comment]
> Hey @dreyfus92, just found out now that we're missing the `ui.ts` label for the "Contributor Guides" item in the sidebar that is were you go to this page, could you add this to the PR? After this LGTM & good to be merged!

do you mean the  `rightSidebar`?

**Comment by Yan-Thomas** [comment]
> do you mean the rightSidebar?

Yes, here's the label: https://github.com/withastro/docs/blob/main/src/i18n/en/ui.ts#L26

**Comment by dreyfus92** [comment]
Done @Yan-Thomas 😄

**Comment by Yan-Thomas** [comment]
Thanks @dreyfus92, LGTM!

**Comment by kevinzunigacuellar** [comment]
Thanks a lot @Yan-Thomas and @dreyfus92 🚀

**Pull request merged**